tests/integration/Migration/Action/GenerateCest.php

```diff
@@ -216,4 +216,68 @@ public function getQuoteWrappedColumns(IntegrationTester $I): void
         $I->assertSame("'column1'", $class->getQuoteWrappedColumns()[0]);
         $I->assertSame("'column2'", $class->getQuoteWrappedColumns()[1]);
     }
+
+    /**
+     * @param IntegrationTester $I
+     */
+    public function throwUnknownColumnTypeException(IntegrationTester $I): void
+    {
+        $I->wantToTest('Migration\Action\Generate - getColumns() throw UnknownColumnTypeException');
+
+        $I->expectThrowable(UnknownColumnTypeException::class, function () {
+            $columns = [
+                new Column('unknown', [
+                    'type' => 9000,
+                    'size' => 10,
+                    'notNull' => true,
+                ]),
+            ];
+
+            $data = [];
+            $class = new Generate('mysql', $columns);
```

Comment:
Can you add PostgreSQL tests and maybe SQLite ?

Reply:
Here it is not DB test, but class(es) logic.

Reply:
But after, create a new Generate instance, you use getColumns(). 
Then getColumns() use getColumnSize() and in this particular function there is a difference between Postgres and Mysql.
`if ($this->adapter === Migration::DB_ADAPTER_POSTGRESQL && in_array($columnType, $noSizePostgres))`
I think it's useful to test it to check if there is no error produce.

Reply:
Added for all 3 DB types.